packages/taquito/signature.json

```diff
@@ -149,7 +149,7 @@
             "sha512": "4241627012300df6926f6f0f3804396d7300697e2cbdccea99e8be56cf9d71c49c0b358c3bc4b61023edcc5226af0402bcd871b052420555d1585d976ae16eb9"
```

Comment:
These signature files are generated automatically, should not be manually modified.

Reply:
okay let me revert these back, I was thinking the same but couldn't find them in gitignore so thought maybe I modify them and they will probably get overwritten anyways.

Reply:
Done